app/main.py

```diff
@@ -1,2 +1,56 @@
+from collections.abc import Hashable
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
+            load_factor: float = 0.75
+    ) -> None:
+        self.load_factor = load_factor
+        self.capacity = initial_capacity
+        self.size = 0
+        self.table = [[] for _ in range(self.capacity)]
```

Comment:
From requirements:

>     Also, don't forget to store (key, hash, value) as a node in the hash table. A hash table may be stored as a simple list of nodes.

Create class Node as per requirements and make a list of Nodes instead of list of lists. Also It is better to fill table with None values on init. It will be more efficient then creating a thousand empty lists.